nodejs/src/utils.js

```diff
@@ -49,7 +49,7 @@ export class Input {
       if (value === '') {
         return defaultValue; // blank entry returns the default
       }
-      for (var idx = 0; idx < one_of_list.length; ++idx) {
+      for (let idx = 0; idx < one_of_list.length; ++idx) {
```

Comment:
perfect usecase for filter 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter

Reply:
Good point. I'd prefer to restrict this PR to eslint-specific changes.